test/TokenPriceDataFeedFactory.test.js

```diff
@@ -0,0 +1,95 @@
+const TokenPriceDataFeedFactory = artifacts.require('TokenPriceDataFeedFactory.sol')
+const TokenPriceDataFeed = artifacts.require('TokenPriceDataFeed.sol')
+const ExchangeAdapterMock = artifacts.require('ExchangeAdapterMock.sol')
+const { expectRevert } = require('./helpers')
+
+const token1 = '0x19a150a4e966bac5bd0473153f5c526d8fa7d4e7'
+const token2 = '0xa27d79db9c02ab01de0ca0f8468cbad2c578f609'
+const token3 = '0xca93ab2dc789c53b062253690ed33ae4eaed4ba3'
+
+contract('TokenPriceDataFeedFactory', () => {
```

Comment:
in this file, sometimes you leave a new line b/w the heading / first line under heading, sometimes you don't. I usually leave no extra line break